src/System.CommandLine.Tests/SuggestionTests.cs

```diff
@@ -137,7 +137,7 @@ public void Command_Suggest_with_text_to_match_orders_by_match_position_then_alp
                 new Command("andmyothersubcommand"),
             };
 
-            var suggestions = command.GetSuggestions("my");
+            var suggestions = command.GetSuggestions(null, "my");
```

Comment:
We can avoid the need to do this by overloading `GetSuggestions` via an extension method. This way we're not introducing a source-breaking change.

Reply:
Fair enough. Would it be ok to make `SymbolExtensions` public and keep the existing extension methods as internal?